Qase_Test/Src/Tests/ApiTests/ProjectApiTests.cs

```diff
@@ -0,0 +1,73 @@
+using System.Net;
+using Allure.Commons;
+using FluentAssertions;
+using NUnit.Allure.Attributes;
+using NUnit.Allure.Core;
+using NUnit.Framework;
+using Qase_Test.Core;
+using Qase_Test.Fakers;
+using Qase_Test.Models;
+using Qase_Test.Steps.ApiSteps;
+
+namespace Qase_Test.Tests.ApiTests
+{
+    [AllureNUnit]
+    [AllureTag("Functional")]
+    [AllureSeverity(SeverityLevel.critical)]
+    [AllureOwner("Bachurin")]
+    [AllureSuite("API")]
+    public class ProjectApiTests
+    {
+        private Project _project;
+        private string _token;
+
+        [SetUp]
+        public void SetUp()
+        {
+            _project = TestDataGeneratorService.GetFakeProject();
+            _token = UserSettings.Token;
+        }
+
+        [Test, Description("Project сreation check")]
+        [AllureSubSuite("ProjectApi")]
+        public void CreateProjectTest()
+        {
+            ProjectApiSteps.CreateProject(_project, _token).Should().Be(HttpStatusCode.OK);
+        }
+
+        [Test, Description("Delete project")]
+        [AllureSubSuite("ProjectApi")]
+        public void DeleteProjectTest()
+        {
+            ProjectApiSteps.CreateProject(_project, _token);
+
+            ProjectApiSteps.DeleteProject(_project, _token).Should().Be(HttpStatusCode.OK);
+        }
+
+        [Test, Description("Create project by member")]
+        [AllureSubSuite("ProjectApi")]
+        public void CreateProjectByMemberTest()
+        {
+            ProjectApiSteps.CreateProject(_project, UserSettings.MemberToken).Should().Be(HttpStatusCode.Forbidden);
```

Comment:
UserSettings.MemberToken according to the previous comment, you should create default parameter for such cases, but  generally you use only _token